src/ChatPage.cpp

```diff
@@ -906,16 +906,22 @@ ChatPage::sendDesktopNotifications(const mtx::responses::Notifications &res)
                                 if (isRoomActive(room_id))
                                         continue;
 
-                                notificationsManager.postNotification(
-                                  room_id,
-                                  QString::fromStdString(event_id),
-                                  QString::fromStdString(cache::singleRoomInfo(item.room_id).name),
-                                  cache::displayName(room_id, user_id),
-                                  utils::event_body(item.event),
-                                  cache::getRoomAvatar(room_id));
+                                if (userSettings_->hasAlertOnNotification()) {
+                                        QApplication::alert(this);
```

Comment:
Do you intentionally use QApplication::alert instead of [QWindow::alert](https://doc.qt.io/qt-5/qwindow.html#alert) here? I'm not sure, what the difference is, they probably do the same thing, but the latter one was introduced more recently. Not that you should change this, I'm just curious, if you considered that and if you know the difference, since I don't. :D

Reply:
I didn't know about `QWindow::alert()`'s existence.
`QApplication::alert()` calls `QWindow::alert()` internally: https://code.qt.io/cgit/qt/qtbase.git/tree/src/widgets/kernel/qapplication.cpp?h=5.15.0#n3899